docs/how-to-docs/drone-ci.md

```diff
@@ -82,15 +83,18 @@ to:
     - promote
 ```
 
-Please note that:
+### Caveats & Known Issues
 
 1. As the `deployment` event has been replaced by the `promote` and `rollback` events, the `drone` CLI no longer supports `drone deploy`.
     Instead, `drone build promote` has to be used to trigger a `promote` event.
 1. If you think you might have to switch to the old pipeline syntax running on the old drone instances during development and testing of your new drone pipeline - for example, if there is a need to apply a fix to a higher environment when your new pipeline is not yet ready, consider specifying a different pipeline file for drone v1.
     In `https://<new-drone-url>/<organisation>/<repository-name>/settings`, you can specify the name of the pipeline file. Drone v0.8 will use `.drone.yml` and for Drone v1, you could specify another file name, e.g. `.drone-v1.yml`. This would allow you to easily and quickly switch between the old and new drone instances and still get the drone servers to use the appropriate pipeline, from the same commit point.
 1. Gitlab supports only one webhook to Drone per repository. So, assuming a repo is active on the old drone instance for gitlab, activating it on the new drone instance will get Gitlab to only send events to the new drone instance (even though the repository will still appear to be active on the old drone instance).
     However, Github supports several webhooks. In that case, simply activating a repo on the new drone instance will not prevent the old drone instance to attempt to run the old pipeline. So make sure you deactivate the repo on the old drone instance unless you really want both pipelines to be active at the same time.
-1. If a repo is activate on a new Drone instance without its pipeline being modified, it will hang and needs to be cancelled.
+1. If a repo is activated on a new Drone instance without its pipeline being modified, it will hang and needs to be cancelled.
```

Comment:
"this is because ..." no runner specified, doesn't know where to send the job?

Reply:
I have no idea about this one - I just fixed a typo - most likely because the pipelines for 0.8 are incompatible with v1. Probably @nefischer can confirm

Reply:
Just fixed this.